index.js

```diff
@@ -60,7 +60,11 @@ const getBorderChars = borderStyle => {
 	return chars;
 };
 
-const getBackgroundColorName = x => camelCase('bg', x);
+// Color must either be a valid chalk color or a color hex ( #55aabb )
+const isHex = color => color.match(/^#[0-f]{6}/i);
```

Comment:
Users will expect shorthands to work, so this needs to take into account `#fff` too.

Reply:
sure